locales/index.js

```diff
@@ -0,0 +1,9 @@
+import {es} from './es'
+import {en} from './en'
+
+const locales = {
+  en,
+  es
```

Comment:
What's the schema for these? I think it should be a [language tag](https://www.w3.org/International/questions/qa-lang-2or3), but we should make that clear.

Reply:
Yes, I can add a comment.

Reply:
does es exist yet?

Reply:
It's just an empty object for now. I've used it to try `polyglot.js` features at the beginning and I included in this PR as I thought that the Spanish translation PR would be merged into this branch and merge all the code at once.